### PR TITLE
mian->main

### PR DESCRIPTION
**Pull request by MercuryDemo**
A common typo when my classmates and I were first learning programming

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -34732,6 +34732,7 @@ mezmorizing->mesmerizing
 miagic->magic
 miagical->magical
 mial->mail
+mian->main
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:34735]: Perhaps `moan` as well?
```suggestion
mian->main, moan,
```

Comment (MercuryDemo) [reply codespell_lib/data/dictionary.txt:34735]: Compared to "main", "moan" doesn't seem to be a commonly used programming word, but it seems to work.
Thanks!

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:34735]: codespell is not used exclusively for code. Not sure what you mean by "it seems to work".

Comment (MercuryDemo) [reply codespell_lib/data/dictionary.txt:34735]: Adding "moan" is a good suggestion, thanks!

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:34735]: Then could you apply the change?

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**